CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Changelog for NeoFS Node
 - FSTree storage now uses more efficient and safe temporary files under Linux (#2566)
 
 ### Removed
+- `neofs-cli container set-eacl` removal of the `no-precheck` flag (#2496)
```

Comment:
The section is named "removed", therefore this line should be more like `deprecated no-precheck flag of neofs-cli container set-eacl`. It's important that this flag was deprecated previously as well.